src/GraphQL.Client.Abstractions.Websocket/GraphQL.Client.Abstractions.Websocket.csproj

```diff
@@ -11,4 +11,11 @@
 	<ProjectReference Include="..\GraphQL.Client.Abstractions\GraphQL.Client.Abstractions.csproj" />
 	</ItemGroup>
 
+	<ItemGroup>
```

Comment:
I recommend to move this dependency inside shared props file instead of copying everywhere.

Reply:
yep, good point